poker-pals/src/components/Login.vue

```diff
@@ -1,14 +1,174 @@
 <template>
-  <div>
-    <h1>Login screen</h1>
-    <p>Content here!</p>
+  <div v-cloak class="container">
+    <div class="row">
+      <div class="col">
+        <h1>Poker Pals</h1>
+        <h3>Play poker with your pals</h3>
+      </div>
+    </div>
+
+    <div class="row align-items-center">
+      <!--  Poker Pals Logo  -->
+      <div class="col-md-6 offset-md-0 col-6 offset-3 ">
+        <img alt="Poker Pals logo" src="../assets/logo.svg">
+      </div>
+
+      <div class="col-md-6">
+        <!-- Login Form -->
+        <form class="text-left" v-on:submit="loginAction" v-if="showLogin">
+          <div class="form-group">
+            <label for="email">Email</label>
+            <input class="form-control" type="email" id="loginEmail" v-model="loginData.email" required/>
+          </div>
+
+          <div class="form-group">
+            <label for="password1">Password</label>
```

Comment:
What is password1? Is it the first of two fields where you need to enter your new password when you're signing up for an account?  I'm not sure if the for field gets used again anywhere but is it worth making password1 and password2 more descriptive?

Reply:
Fields are not used anywhere, I'll remove them